_posts/2017-06-13-troubleshooting.adoc

```diff
@@ -10,6 +10,33 @@ toc::[]
 
 == Build problems
 
+=== Bitbake builds fail on Ubuntu 18.0.4
```

Comment:
This doesn't seem like a good title. There are a ton of reasons a build might fail, apart from this particular java error. Maybe something like "Build fails with a java.security.InvalidAlgorithmParameterException error"?